src/js/i18n/en-US.yml

```diff
@@ -241,6 +241,8 @@ careOptsFrontend:
           patient: Patient
           results: Results
           workspaceStatus: Status
+        itemView:
+          showMatchData: "{label, select, Name {} Birth_Date {} other {true}}"
```

Comment:
Ideally we could just do something like this:

```js
showMatchData: "{label, select, Name {} Birth_Date {} other {value}}"
```

But I don't know of a way to have the UI display nothing if the label is `Name` or `Birth_Date`. for `Name {}`/`Birth_Date {}`, `undefined` is displayed in the UI. And doing `{''}` shows the actual quotes in the UI.

So instead I have this returning either `undefined` or `"true"`. And then the view template determines if the `match.value`/`match.label` should be displayed.

Reply:
For the `Birth_Date` underscore. I had to normalize the spacing in the labels. Since having the `Birth Date` in there with the space broke things.